modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -6065,11 +6065,12 @@
             <c:list-property name="connectors" required="false" readOnly="true" description="Specifies the names of connectors that will be broadcast." >
               <c:simple-property name="connectors" />
             </c:list-property>
-            <c:simple-property name="socket-binding" required="false" type="string" readOnly="true" description="The broadcast group socket binding. Alternative to jgroups-channel">
+            <c:simple-property name="socket-binding" required="false" type="string" readOnly="true" description="The broadcast group socket binding. Alternative to jgroups-channel. With EAP 7.2+ use jgroups-cluster attribute instead for configuration.">
               <c:option-source target="configuration" expression="*/socket-binding=name:type=SocketBindingGroup" expressionScope="baseResource"/>
             </c:simple-property>
-            <c:simple-property name="jgroups-channel" required="false" type="string" readOnly="true" description="The name used by a JGroups channel to join a cluster. Alternative to socket-binding."/>
-            <c:simple-property name="jgroups-stack" required="false" type="string" readOnly="true" description="The name of a stack defined in the org.jboss.as.clustering.jgroups subsystem that is used to form a cluster. Requires jgroups-channel"/>
+            <c:simple-property name="jgroups-cluster" required="false" type="string" readOnly="true" description="The logical cluster name. Introduced in EAP 7.2. Alternative to Socket Binding. Before EAP 7.2 this attribute is silently ignored"/>
+            <c:simple-property name="jgroups-channel" required="false" type="string" readOnly="true" description="References the name of a JGroups channel. With EAP 7.2+: If undefined, the default channel will be used else jgroups-cluster will be required."/>
```

Comment:
Should probably make these consistent.

Also, is this just added to 7.2? Or has it been there since 7.0? And if its been there since 7.0, did 7.2 introduce the requirement for jgroups-cluster when jgroups-channel is set?

If just added in 7.2, then:
```xml
            <c:simple-property name="jgroups-channel" required="false" type="string" readOnly="true" description="EAP 7.2+: References the name of a JGroups channel. If undefined, the default channel will be used otherwise jgroups-cluster will also be required. Prior to EAP 7.2: This attribute is silently ignored."/>
```
If always been there since 7.0 and 7.2 introduced the requirement for jgroups-cluster when jgroups-channel is set:
```xml
            <c:simple-property name="jgroups-channel" required="false" type="string" readOnly="true" description="References the name of a JGroups channel. If undefined, the default channel will be used. EAP 7.2+: If defined, jgroups-cluster will also be required."/>
```

Reply:
The later.
That property was an alternative to socket-binding prior EAP 7.2.

Reply:
I'm moving it a bit, the "If undefined, the default channel will be used", because this only applies to EAP 7.2+

```xml
<c:simple-property name="jgroups-channel" required="false" type="string" readOnly="false" description="References the name of a JGroups channel. EAP 7.2+: If undefined, the default channel will be used. If defined, jgroups-cluster will also be required."/>
```